vogue/commands/load/application_tag.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import click
+import json
+from vogue.load.application_tag import load_aplication_tags
+
+
+LOG_LEVELS = ['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL']
+LOG = logging.getLogger(__name__)
+
+
+@click.command("apptag", short_help="Reads database json dump with application tag info.")
+@click.option('-d', '--db-dump', type=str, required=True, help='Database dump in json')
```

Comment:
Coming from `cg` doesn't make it a db dump, so I would suggest to name it according to the information that is contained in the json object.

A mandatory single option is an argument ;)

```suggestion
@click.argument('application-tags', type=str, help='json formatted application tags')
```

Reply:
`click.option` by default is `type=str`, there is no need to set the type imho.

Reply:
👍

Reply:
Im all new to click, but giving this as argument instead of option doesn't seem to work.